pkg/router/controller/router_controller.go

```diff
@@ -138,7 +139,7 @@ func (c *RouterController) UpdateNamespaces() {
 	//       cleanup issues as old == new in Plugin.HandleNamespaces().
 	namespaces := sets.NewString(c.FilteredNamespaceNames.List()...)
 
-	glog.V(4).Infof("Updating watched namespaces: %v", namespaces)
+	log.V(4).Info("updating watched namespaces", "namespaces", namespaces)
```

Comment:
`namespaces.List()` would print more prettily (but we can save that for a separate clean-up).